bindings/python/src/lib.rs

```diff
@@ -262,25 +412,47 @@ impl safe_open {
 
         let data = &self.mmap[info.data_offsets.0 + self.offset..info.data_offsets.1 + self.offset];
 
-        let array: PyObject = Python::with_gil(|py| PyByteArray::new(py, data).into_py(py));
-
-        create_tensor(
-            &self.framework,
-            info.dtype,
-            &info.shape,
-            array,
-            &self.device,
-        )
+        match (&self.device, &self.framework, &self.cudart) {
+            (Device::Cuda(_), Framework::Pytorch, Some(cudart)) => {
+                Python::with_gil(|py| -> PyResult<PyObject> {
+                    let module = PyModule::import(py, intern!(py, "torch"))?;
```

Comment:
Do we need to do this import every time we get a tensor ?

Reply:
You can't use statics for the same reason is that all Python variables have a python lifetime associated with it.

However, I just discovered : https://docs.rs/pyo3/latest/pyo3/once_cell/struct.GILOnceCell.html !